include/translatable-object.php

```diff
@@ -324,8 +324,6 @@ public function join_clause( $alias = '' ) {
 	 *
 	 * @param PLL_Language|PLL_Language[]|string|string[] $lang A `PLL_Language` object, or a comma separated list of language slugs, or an array of language slugs or objects.
 	 * @return string The WHERE clause.
-	 *
-	 * @phpstan-param PLL_Language|PLL_Language[]|non-empty-string|non-empty-string[] $lang
```

Comment:
Why remove this?

Reply:
In fact, it seems to me that the me that the method can work with empty strings.

Reply:
Indeed.